src/ManifestDownloadLinks.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Link from '@material-ui/core/Link';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Typography from '@material-ui/core/Typography';
+
+/**
+ * ManifestDownloadLinks ~
+*/
+export default class ManifestDownloadLinks extends Component {
+  renderings() {
+    const { manifest } = this.props;
+
+    return manifest.getSequences()[0].getRenderings();
+  }
+
+
+  /**
+   * Returns the rendered component
+  */
+  render() {
+    if (this.renderings().length < 1) {
+      return '';
+    }
+
+    return (
+      <span>
+        <Typography variant="h3">Other download options</Typography>
+        <List>
+          {this.renderings().map(rendering => (
+            <ListItem disableGutters divider key={rendering.id}>
+              <Link href={rendering.id} rel="noopener noreferrer" target="_blank" variant="body1">
+                {rendering.getLabel().map(label => label.value)[0]}
+              </Link>
+              {(rendering.getFormat() && rendering.getFormat().value)
+                && (
+                  <ListItemText primaryTypographyProps={{ variant: 'body1' }}>
+                    {`(${rendering.getFormat().value})`}
+                  </ListItemText>
+                )
+              }
+            </ListItem>
+          ))}
+        </List>
+      </span>
+    );
+  }
+}
+
+ManifestDownloadLinks.propTypes = {
+  manifest: PropTypes.shape({
```

Comment:
Do we anticipate needing other things out of `manifest`, or should the component just get the list of renderings directly?